pandas/io/parquet.py

```diff
@@ -269,6 +269,10 @@ def read_parquet(path, engine="auto", columns=None, **kwargs):
         expected. A local file could be:
         ``file://localhost/path/to/table.parquet``.
 
+        A file path can also be a directory name that contains multiple(partitioned)
```

Comment:
I think this would benefit from an example. It's not clear to me what happens based on this description alone.

Does this apply to either parquet engine?